### ISSUE TITLE
Absent Bell's phenomenon

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#9644]
**Preferred term label:**

**Synonyms**
 palpebral oculogyric reflex

**Definition (free text, please give PubMed ID)**
Bell's phenomenon A protective effect seen in [BELL'S PALSY](https://medical-dictionary.thefreedictionary.com/Bell%27s+palsy). The attempt to close the eyes tightly causes the exposed eye on the paralysed side of the face to roll upwards. This protects the eye by helping to keep the cornea moist.
Collins Dictionary of Medicine © Robert M. Youngson 2004, 2005

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Congenital fibrosis of the extraocular muscles (CFEOM) **ORPHA 45358**

**Your nano-attribution (ORCID)**

PMID: 16815872 "Three patients developed minimal, unsustained abduction of an eye during attempted down gaze (Patients 4, 5 and 15). Bell’s phenomenon was absent in all patients."

PMID: 36780505 "We also analyzed the materials used for frontalis  slings. Of the 56 frontalis sling operations, 42 used autologous fascia lata, 12 used silicone rods, and 2 used stored fascia lata. Patients with absent Bell’s phenomenon are more likely to experience corneal complications than those with proper elevation of the eye  with lid closure"

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6000709
- label: Absent Bell phenomenon
- parents: [Abnormal involuntary eye movements (HP:0012547)](https://hpo.jax.org/app/browse/term/HP:0012547)
- pmid: PMID:16815872|PMID:36780505
- def: The Bell phenomenon is the reflex upward movement of the front of the eyeball when the eyelids close (or blink). It protects the eye by helping to keep the cornea moist. This term refers to a lack of this phenomenon, which can be observed with certain conditions such as congenital fibrosis of the extraocular muscles.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9644
- synonyms: Absent palpebral-oculogyric reflex(NONE)
